blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/BitbucketCredentialUtils.java

```diff
@@ -0,0 +1,22 @@
+package io.jenkins.blueocean.blueocean_bitbucket_pipeline;
+
+import com.google.common.base.Preconditions;
+import io.jenkins.blueocean.blueocean_bitbucket_pipeline.cloud.BitbucketCloudScm;
+import io.jenkins.blueocean.blueocean_bitbucket_pipeline.server.BitbucketServerScm;
+import org.apache.commons.codec.digest.DigestUtils;
+
+/**
+ * @author cliffmeyers
+ */
+public class BitbucketCredentialUtils {
+
+    public static String computeCredentialId(String scmId, String apiUrl) {
```

Comment:
Fixed in 571f3891639a1962430cc0a428ee481eb2280fa5. Included similar logic for GitHub in fix dde0eb17dfbd8939490a3e3fea2618dbbe6f4f55